### ISSUE TITLE
Can't build on Linux Mint 17.2

### ISSUE THREAD

**Issue by kowalcj0** [arybczak/ncmpcpp#51]
I've installed all dependencies, but `make` fails to build the binary

```
make[2]: Entering directory `/home/kowalcj0/git/ncmpcpp/src'
g++ -DHAVE_CONFIG_H -I. -I..   -I/usr/include/x86_64-linux-gnu   -I/usr/local/include   -I/usr/include/ncursesw  -g -O2 -std=c++0x -MT comparators.o -MD -MP -MF .deps/comparators.Tpo -c -o comparators.o `test -f 'utility/comparators.cpp' || echo './'`utility/comparators.cpp
mv -f .deps/comparators.Tpo .deps/comparators.Po
g++ -DHAVE_CONFIG_H -I. -I..   -I/usr/include/x86_64-linux-gnu   -I/usr/local/include   -I/usr/include/ncursesw  -g -O2 -std=c++0x -MT html.o -MD -MP -MF .deps/html.Tpo -c -o html.o `test -f 'utility/html.cpp' || echo './'`utility/html.cpp
mv -f .deps/html.Tpo .deps/html.Po
g++ -DHAVE_CONFIG_H -I. -I..   -I/usr/include/x86_64-linux-gnu   -I/usr/local/include   -I/usr/include/ncursesw  -g -O2 -std=c++0x -MT option_parser.o -MD -MP -MF .deps/option_parser.Tpo -c -o option_parser.o `test -f 'utility/option_parser.cpp' || echo './'`utility/option_parser.cpp
mv -f .deps/option_parser.Tpo .deps/option_parser.Po
g++ -DHAVE_CONFIG_H -I. -I..   -I/usr/include/x86_64-linux-gnu   -I/usr/local/include   -I/usr/include/ncursesw  -g -O2 -std=c++0x -MT string.o -MD -MP -MF .deps/string.Tpo -c -o string.o `test -f 'utility/string.cpp' || echo './'`utility/string.cpp
mv -f .deps/string.Tpo .deps/string.Po
g++ -DHAVE_CONFIG_H -I. -I..   -I/usr/include/x86_64-linux-gnu   -I/usr/local/include   -I/usr/include/ncursesw  -g -O2 -std=c++0x -MT type_conversions.o -MD -MP -MF .deps/type_conversions.Tpo -c -o type_conversions.o `test -f 'utility/type_conversions.cpp' || echo './'`utility/type_conversions.cpp
mv -f .deps/type_conversions.Tpo .deps/type_conversions.Po
g++ -DHAVE_CONFIG_H -I. -I..   -I/usr/include/x86_64-linux-gnu   -I/usr/local/include   -I/usr/include/ncursesw  -g -O2 -std=c++0x -MT wide_string.o -MD -MP -MF .deps/wide_string.Tpo -c -o wide_string.o `test -f 'utility/wide_string.cpp' || echo './'`utility/wide_string.cpp
mv -f .deps/wide_string.Tpo .deps/wide_string.Po
g++ -DHAVE_CONFIG_H -I. -I..   -I/usr/include/x86_64-linux-gnu   -I/usr/local/include   -I/usr/include/ncursesw  -g -O2 -std=c++0x -MT actions.o -MD -MP -MF .deps/actions.Tpo -c -o actions.o actions.cpp
actions.cpp: In constructor ‘Actions::UpdateEnvironment::UpdateEnvironment()’:
actions.cpp:315:10: error: ‘from_time_t’ is not a member of ‘boost::posix_time’
 , m_past(boost::posix_time::from_time_t(0))
          ^
make[2]: *** [actions.o] Error 1
make[2]: Leaving directory `/home/kowalcj0/git/ncmpcpp/src'
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory `/home/kowalcj0/git/ncmpcpp'
make: *** [all] Error 2
```

**Comment by arybczak** [comment]
Does adding `#include <boost/date_time/posix_time/posix_time.hpp>` to the list of includes in actions.cpp solve the problem?

**Comment by kowalcj0** [comment]
yes it did. Cheers

**Comment by arybczak** [comment]
Fixed by 4546d5477770a92fe107cca7a060ec7a57a542e5
